Cleanup pytest warnings and errors on chroot platform

**What happened**:
As of now, there are some warnings and errors as soon as pytests are executed on the chroot platform. 

This includes warnings like:
```
helper/tests/password_shadow.py:53
  /gardenlinux/tests/helper/tests/password_shadow.py:53: DeprecationWarning: invalid escape sequence '\*'
    "awk -F: '$2~/\*/ { next; } $2~/!/ { next; } {print $0; exit 1}' /etc/shadow", quiet=True)
...
/usr/local/lib/python3.10/dist-packages/paramiko/transport.py:453: DeprecationWarning: setDaemon() is deprecated, set the daemon attribute instead
    self.setDaemon(True)
...
/usr/local/lib/python3.10/dist-packages/paramiko/util.py:235: DeprecationWarning: currentThread() is deprecated, use current_thread() instead
    tid = id(threading.currentThread())
...
/usr/local/lib/python3.10/dist-packages/paramiko/buffered_pipe.py:104: DeprecationWarning: notifyAll() is deprecated, use notify_all() instead
    self._cv.notifyAll()
```

And errors like:
```
def test_loadavg(client, non_kvm):
        """This test does not produce any load. Make sure no
           other process does."""
        (exit_code, output, error) = client.execute_command("cat /proc/loadavg")
>       assert exit_code == 0, f"Expected to be able to show contents of /proc/loadavg"
E       AssertionError: Expected to be able to show contents of /proc/loadavg
E       assert 1 == 0
E         +1
E         -0

client     = <helper.sshclient.RemoteClient object at 0x7f058e1e05b0>
error      = 'cat: /proc/loadavg: No such file or directory\n'
exit_code  = 1
non_kvm    = None
output     = ''
```
```
def test_selinux(client):
        (exit_code, output, error) = client.execute_command("grep selinux /sys/kernel/security/lsm")
>       assert exit_code == 0, f"no {error=} expected"
E       AssertionError: no error='grep: /sys/kernel/security/lsm: No such file or directory\n' expected
E       assert 2 == 0
E         +2
E         -0

client     = <helper.sshclient.RemoteClient object at 0x7f058e1e05b0>
error      = 'grep: /sys/kernel/security/lsm: No such file or directory\n'
exit_code  = 2
output     = ''
```

**What you expected to happen**:
There shouldn't be warnings and errors

**How to reproduce it (as minimally and precisely as possible)**:
Execute the following command:
```
./build.sh --skip-build --lessram --features server,cloud,kvm,_dev .build dev
```

**Anything else we need to know**:
Some warnings can be fixed by updating `paramiko` from `2.10.1` to `2.10.4`. The newest `paramiko` version removes these deprecation warnings that came up with Python 3.10.

The errors on the other hand seems to not work properly on the chroot platform. Therefore, they probably need to be disabled via the `non_chroot` fixture.

--- a/tests/helper/tests/password_shadow.py
+++ b/tests/helper/tests/password_shadow.py
@@ -50,12 +50,12 @@ class PasswordShadow():
             cls.instance = super(PasswordShadow, cls).__new__(cls)
 
             (exit_code, output, error) = client.execute_command(
-                "awk -F: '$2~/\*/ { next; } $2~/!/ { next; } {print $0; exit 1}' /etc/shadow", quiet=True)
+                "awk -F: '$2~/\\*/ { next; } $2~/!/ { next; } {print $0; exit 1}' /etc/shadow", quiet=True)
             if exit_code != 0:
                 raise TestFailed(f"No passwords should be set in /etc/shadow")
 
             (exit_code, output, error) = client.execute_command(
-                "awk -F: '$2~/\*/ { next; } $2~/x/ { next; } {print $0; exit 1}' /etc/passwd", quiet=True)
+                "awk -F: '$2~/\\*/ { next; } $2~/x/ { next; } {print $0; exit 1}' /etc/passwd", quiet=True)
             if exit_code != 0:
                 raise TestFailed(f"Malformed entries in /etc/passwd \n {output} {error}")
 

--- a/tests/helper/tests/sgid_suid_files.py
+++ b/tests/helper/tests/sgid_suid_files.py
@@ -12,7 +12,7 @@ def _get_remote_files(client, perm):
     """ Get remote files of given attribute """
     files = []
     cmd_find = f"find / -type f -perm {perm} -exec "
-    cmd_stat = "stat -c '%n,%u,%g' {} \; 2> /dev/null"
+    cmd_stat = "stat -c '%n,%u,%g' {} \\; 2> /dev/null"
     cmd = cmd_find + cmd_stat
     (exit_code, output, error) = client.execute_command(
         cmd, quiet=True)

--- a/tests/integration/test_gardenlinux.py
+++ b/tests/integration/test_gardenlinux.py
@@ -118,7 +118,7 @@ def test_timesync(client, azure):
     (exit_code, output, error) = client.execute_command("test -L /dev/ptp_hyperv")
     assert exit_code == 0, f"Expected /dev/ptp_hyperv to be a symbolic link"
 
-def test_loadavg(client, non_kvm):
+def test_loadavg(client, non_kvm, non_chroot):
     """This test does not produce any load. Make sure no 
        other process does."""
     (exit_code, output, error) = client.execute_command("cat /proc/loadavg")
@@ -337,7 +337,7 @@ def test_apparmor(client, non_chroot):
     (exit_code, output, error) = client.execute_command("grep apparmor /sys/kernel/security/lsm")
     assert exit_code == 1, f"expected apparmor to be disabled"
 
-def test_selinux(client):
+def test_selinux(client, non_chroot):
     (exit_code, output, error) = client.execute_command("grep selinux /sys/kernel/security/lsm")
     assert exit_code == 0, f"no {error=} expected"
     assert "selinux" in output.rstrip(), "Expected SELinux to be enabled."

